addons/l10n_it_edi_sdicoop/models/res_config_settings.py

```diff
@@ -8,10 +8,13 @@ class ResConfigSettings(models.TransientModel):
 
     is_edi_proxy_active = fields.Boolean(compute='_compute_is_edi_proxy_active')
 
-    @api.depends('company_id.account_edi_proxy_client_id')
+    @api.depends('company_id.account_edi_proxy_client_ids')
     def _compute_is_edi_proxy_active(self):
         for config in self:
-            config.is_edi_proxy_active = config.company_id.account_edi_proxy_client_id
+            config.is_edi_proxy_active = config.company_id.account_edi_proxy_client_ids
 
     def button_create_proxy_user(self):
-        self.company_id._create_proxy_user()
+        # For now, only fattura_pa uses the proxy.
+        # To use it for more, we have to either make the activation of the proxy on a format basis
+        # or create a user per format here (but also when installing new formats)
+        self.env.ref['l10n_it_edi.fattura_pa']._create_proxy_user(self.company_id)
```

Comment:
Is there an object for this?  Would indeed be good to have it private on the object itself.

Reply:
Yeah, I simplified that.